src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml

```diff
@@ -1,16 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Settings xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.microsoft.com/2011/01/fabric">
-  <!-- This is used by the StateManager's replicator. -->
-  <Section Name="ReplicatorConfig">
-    <Parameter Name="ReplicatorEndpoint" Value="ReplicatorEndpoint" />
+  <!-- Add your custom configuration sections and parameters here -->
+  <!--
+  <Section Name="MyConfigSection">
+    <Parameter Name="MyParameter" Value="Value1" />
   </Section>
-  <!-- This is used for securing StateManager's replication traffic. -->
-  <Section Name="ReplicatorSecurityConfig" />
-
-  <!-- Add your custom configuration sections and parameters here. -->
-  
-  <Section Name="SubscriberConfiguration">
```

Comment:
why did you delete it? it's the way, how to pass, Topic name to Subscriber via configuration file